gemma-web/src/main/java/ubic/gemma/web/controller/IndexController.java

```diff
@@ -25,10 +23,7 @@
 @Controller
 public class IndexController {
 
-    @Autowired
-    private IndexService indexService;
-
     public String index( IndexerTaskCommand command ) {
-        return this.indexService.index( command );
+        return null;
```

Comment:
It would be nice to remove this altogether and remove the frontend parts as well.

Reply:
It wouldn't be too difficult to reuse the logic from the CLI here. Even isolate all this in what used to be the IndexerService and reuse that for both GUI and CLI.